libp2p/protocols/identify.nim

```diff
@@ -88,13 +86,13 @@ proc encodeMsg(peerInfo: PeerInfo, observedAddr: Opt[MultiAddress], sendSpr: boo
 
   let pkey = peerInfo.publicKey
 
-  result.write(1, pkey.getBytes().get())
+  result.write(1, pkey.getBytes().expect("valid key"))
```

Comment:
Do we wanna raise a `Defect` here?

Reply:
For now, yes (the user should provide a valid key)